CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Snyk Security - Code and Open Source Dependencies Changelog
```

Comment:
Do we want to keep changelog?

"Snyk Security Changelog"

Reply:
yes, let me do it and update other plugins accordingly

Reply:
done